test/e2e/common/configmap_volume.go

```diff
@@ -33,18 +33,18 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 	f := framework.NewDefaultFramework("configmap")
 
 	/*
-		    Testname: configmap-nomap-simple
-		    Description: Make sure config map without mappings works by mounting it
-			to a volume with a custom path (mapping) on the pod with no other settings.
+		Release : v1.9
+		Testname: ConfigMap without mapping
+		Description: Create a configMap, create a Pod that mounts a volume and populates the volume with data stored in the configMap. The config map that is created MUST be accessible to read from the newly created Pod using the volume mount. The data content of the file MUST be readable and verified and file modes MUST default to 0x644.
 	*/
 	framework.ConformanceIt("should be consumable from pods in volume [NodeConformance]", func() {
 		doConfigMapE2EWithoutMappings(f, 0, 0, nil)
 	})
 
 	/*
-		    Testname: configmap-nomap-default-mode
-		    Description: Make sure config map without mappings works by mounting it
-			to a volume with a custom path (mapping) on the pod with defaultMode set
+		Release : v1.9
+		Testname: ConfigMap without mapping, volume mode set
+		Description: Create a configMap, create a Pod that mounts a volume and populates the volume with data stored in the configMap. File mode is changed to a custom value of '0x400'. The config map that is created MUST be accessible to read from the newly created Pod using the volume mount. The data content of the file MUST be readable and verified and file modes MUST be set to the custom value of ‘0x400’
```

Comment:
"File mode is changed" ... file mode of what?

nit: you quote '0x400' here but don't quote 0x644 elsewhere, which is it?

Reply:
default value is 644 but the test set it to custom value of 400. It is stated that the file mode set to 400.